docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,44 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+OpenShift provides an integrated docker registry which can be used as a local registry.
+
+This helps to speed up the local development as images present in registry can be used for applications running in OpenShift.
+
+== Login to the docker registry 
+
+. Start minishift with the link:../command-ref/minishift_start{outfilesuffix}[`minishift start`] command.
+
+. Add the oc binary to the PATH environment variable.
+
+ $ export PATH=$PATH:~/.minishift/cache/oc/{openshift-version}
+
+. Export the environment variables that are required link:../using/reusing-docker-daemon{outfilesuffix}[to reuse the Docker daemon]
+
+ $ eval $(minishift docker-env)
```

Comment:
It is more than just one command. All three commands are the same. So by just referencing by saying: "Provided you console is configured to use the Minishift Docker daemon (see bla) we can skip to repeat and maintain the identical steps at two places. If a user knows what "reusing the Minishfit Docker daemon means", all is good, otherwise it makes sense imo top start there.